x-pack/solutions/observability/plugins/slo/server/routes/slo/inspect_slo.ts

```diff
@@ -39,6 +40,10 @@ export const inspectSLORoute = createSloServerRoute({
     const esClient = core.elasticsearch.client.asCurrentUser;
     const username = core.security.authc.getCurrentUser()?.username!;
     const soClient = core.savedObjects.client;
+    const [coreStart] = await corePlugins.getStartServices();
+    const internalSoClient = new SavedObjectsClient(
+      coreStart.savedObjects.createInternalRepository()
+    );
```

Comment:
one nice refactor you could do is move these common declarations like internalSOClinet to https://github.com/elastic/kibana/blob/main/x-pack/solutions/observability/plugins/slo/server/routes/register_routes.ts#L30

this it would become available in `const sloContext = await context.slo;`
and you could pass that context to each service, that way you won't have to declare stuff like this in each route where required.